pyfar/signals/signals.py

```diff
@@ -229,13 +240,13 @@ def pulsed_noise(n_pulse, n_pause, n_fade=90, repetitions=5, amplitude=1,
         The default is 90, which equals approximately 2 ms at sampling rates of
         44.1 and 48 kHz.
     repetitions : int, optional
-    amplitude : double, array like, optional
+    rms : double, array like, optional
         The RMS amplitude of the white noise signal. A multi channel noise
         signal is generated if an array of amplitudes is passed. The default
         is 1.
```

Comment:
Pulsed noise does not support multiple channels
```suggestion
        The route mean square (RMS) value of the noise signal.
```